packages/client/src/utils.ts

```diff
@@ -0,0 +1,89 @@
+import { AES, enc, SHA256 } from 'crypto-js';
+
+export enum StorageType {
+  LocalStorage,
+  IndexedDB,
+  ChromeExtension,
+}
+
+export enum DataType {
+  Key,
+  Message
+}
+
+export function encryptMessage(message: string, key: string): string {
+  return AES.encrypt(message, key).toString();
+}
+
+export function decryptMessage(encryptedMessage: string, key: string): string {
+  return AES.decrypt(encryptedMessage, key).toString(enc.Utf8);
+}
+
+export function generateRandomKey(): string {
+  const random = (Math.random() * 100000000).toString();
+  const date = new Date().toString();
+  return SHA256(random + date).toString();
+}
+
+export function dataTypeToString(dataType: DataType): string {
+  switch (dataType) {
+    case DataType.Key:
+      return 'Key';
+    case DataType.Message:
+      return 'Message';
+    default:
+      throw new Error('Unreachable code');
+  }
+}
```

Comment:
Since only values of the specified standard are used as input, 
they are classified using enums rather than as string values and converted to strings when used.